.github/workflows/publish.yaml

```diff
@@ -12,23 +12,29 @@ jobs:
   npm:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
-      - name: Run ./ci/steps/publish-npm.sh
-        uses: ./ci/images/debian10
+      - uses: actions/checkout@v2
+
+      - name: Download npm package
+        uses: actions/download-artifact@v2
         with:
-          args: ./ci/steps/publish-npm.sh
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          name: npm-package
```

Comment:
Took a second look and need to fix this; `npm-package` is produced by the `ci.yaml` workflow, we need to fetch it from there, this won't work.